core/tests/webdriverio_utils/ContributorDashboardPage.js

```diff
@@ -52,6 +56,18 @@ var ContributorDashboardPage = function() {
     '.e2e-test-question-suggestion-review-modal-header');
   var usernameContainer = $('.e2e-test-username');
   var reviewRightsDiv = $('.e2e-test-review-rights');
+  var selectorContainer = $('.e2e-test-language-selector');
+
+  var _openLanguageSelector = async function() {
+    await action.click('Test Language Selector Container', selectorContainer);
+  };
+
+  var _selectLanguage = async function(language) {
+    await _openLanguageSelector();
+    var selectorOption = selectorContainer.$(
+      `.e2e-test-language-selector-option=${language}`);
+    await action.click('Test Language Selector Option', selectorOption);
```

Comment:
Ditto*

Reply:
Add `language` value in the decsrition, helps with debugging i.e, the error message will clearly show which language was attempted for selection

Reply:
Done.